pandas/tests/window/moments/test_moments_rolling.py

```diff
@@ -1477,6 +1476,7 @@ def test_rolling_consistency_std(consistency_data, window, min_periods, center):
     )
 
 
+@pytest.mark.slow
```

Comment:
@jbrockmendel 
thanks for your comment in https://github.com/pandas-dev/pandas/pull/33813#discussion_r419027086 , indeed, i tested the execution time and seems around 40s which is quite long and should mark as slow. 

since this change is very tiny, i think it is okay to just change it here in this PR although it is out of scope of this PR.